### ISSUE TITLE
vue/attributes-order false nagative for v-bind="object"

### ISSUE THREAD

**Issue by ztyzbb** [vuejs/eslint-plugin-vue#1433]
**Checklist**

- [x] I have tried restarting my IDE and the issue persists.
- [x] I have read the [FAQ](https://eslint.vuejs.org/user-guide/#faq) and my problem is not listed.

**Tell us about your environment**

- **ESLint version: 7.19.0** 
- **eslint-plugin-vue version: 7.5.0** 
- **Node version: 14.9.0** 
- **Operating System: Windows 10 64bit 20H1** 

**Please show your full configuration:**

Very long, only related part, I didn't modify this rule individually. So I'm using vue/attributes-order's defalut order
```json5
{
  extends: [
    'eslint:recommended',
    'plugin:import/recommended',
    'plugin:promise/recommended',
    'plugin:vue/recommended',
  ]
}
```

**What did you do?**	

Very long, only related part
```vue
<template>
  <pick-table
    ref="pickTable"
    v-model="showPickTable"
    v-bind="pickTableBind"
    @picked="picked"/>
</template>
```

**What did you expect to happen?**
No warning

**What actually happened?**

ESLint reported:
warning  Attribute "v-bind" should go before "v-model"  vue/attributes-order

**Repository to reproduce this issue**
I can create one if needed, but I think it's clear enough :)

**Where probably wrong**
#1391 
[getPositionFromAttrIndex function](https://github.com/vuejs/eslint-plugin-vue/blob/5129cef626ea4d69ab7e022869cc22aa9d8409bd/lib/rules/attributes-order.js#L330)
isVAttributeOrVBind() return true for ref attribute, maybe call isVBind() better?
@ota-meshi
